### PR TITLE
Prepare release 1.23.2

### LINKED ISSUE #6428: Point release for Kotlin 1.9.10 support
## Expected Behavior
Can build with Kotlin 1.9.10

## Current Behavior
1.23.1 only supports Kotlin 1.9.0

## Context
Hello, would it be possible to get a point release with Kotlin 1.9.10 support? This is currently blocking our projects from upgrading to Kotlin 1.9.10. ~~Kotlin 1.9.10 has JDK 21 support and we plan on migrating our projects to JDK 21 once Gradle and detekt support it.~~
I along with the community would be very appreciative if we could get a point release with minimal changes quickly, that will allow us to run our Kotlin 1.9.10 builds with detekt. 
Thank you for your support with the last issue https://github.com/detekt/detekt/issues/6278 to release a build to support Kotlin 1.9.0

### PR DESCRIPTION
**Pull request by cortinico**
Closes #6428

A new point release for the 1.x branch is long overdue!

### CODE REVIEW

**Comment by cortinico** [comment]
@detekt/maintainers can we please merge this before we start merging other PRs?

File: website/src/pages/changelog.md [hunk 1]
```diff
@@ -6,6 +6,37 @@ keywords: [changelog, release-notes, migration]
...
+
+- Update kotlin monorepo to v1.9.10 - [#6423](https://github.com/detekt/detekt/pull/6423)
+- Update dependency gradle to v8.4 - [#6522](https://github.com/detekt/detekt/pull/6522)
+- Update dependency gradle to v8.3 - [#6406](https://github.com/detekt/detekt/pull/6406)
+- Update com.android.tools.build to 8.1.1 - [#6419](https://github.com/detekt/detekt/pull/6419)
+- update dependency org.jetbrains.dokka to v1.9.10 - [#6549](https://github.com/detekt/detekt/pull/6549)
+
+##### Contributors
+
+We would like to thank the following contributors that made this release possible: @3flex, @BraisGabin, @ErdoganSeref, @Goooler, @Hexcles, @LeoColman, @PoisonedYouth, @TWiStErRob, @VirtualParticle, @arturbosch, @atulgpt, @bric3, @chao2zhang, @cortinico, @dzirbel, @eygraber, @kkocel, @lexa-diky, @marschwar, @matejdro, @mdemianova, @pablobaxter, @rmarquis, @segunfamisa, @severn-everett, @t-kameyama
```
Comment (3flex) [inline website/src/pages/changelog.md:38]: Not that these users haven't contributed over various releases, but the list of PRs isn't as long as this list, so it doesn't look right... Something to look at next time, it's not a blocker.

**Review by 3flex — APPROVED** [review]

File: website/src/pages/changelog.md (hunk 1 above)
Comment (TWiStErRob) [reply website/src/pages/changelog.md:38]: Could it be that PR authors AND reviewers/commenters/issue openers are also included?

Comment (cortinico) [reply website/src/pages/changelog.md:38]: Nope the filtering was wrong:
- https://github.com/detekt/detekt/pull/6580

**Pull request merged**